tests/test_parameter.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```

Comment:
Added a test for the `parameter` module. There are some things in here that should be fixed.